analyses/focal-cn-file-preparation/07-consensus-annotated-merge.R

```diff
@@ -71,18 +71,18 @@ merged_x_and_y <- rbind(cnvkit_x_and_y, consensus_x_and_y)
 
 # write out the files
 readr::write_tsv(merged_auto, 
-          file.path(opt$outdir, "consensus_wgs_plus_cnvkit_wxs_autosomes.tsv.gz"))
+          file.path(opt$outdir, "consensus_wgs_plus_cnvkit_wxs_plus_freec_tumor_only_autosomes.tsv.gz"))
 
 readr::write_tsv(merged_x_and_y, 
-          file.path(opt$outdir, "consensus_wgs_plus_cnvkit_wxs_x_and_y.tsv.gz"))
+          file.path(opt$outdir, "consensus_wgs_plus_cnvkit_wxs_plus_freec_tumor_only_x_and_y.tsv.gz"))
 
```

Comment:
I don't see that tumor only free c was added above near lines 69-70 like the wgs consensus and wxs cnvkit..